edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Amqp/EdgeHubTlsTransport.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.Azure.Devices.Edge.Hub.Amqp
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net.Security;
+    using System.Security.Cryptography.X509Certificates;
+    using Microsoft.Azure.Amqp.Transport;
+    using Microsoft.Azure.Amqp.X509;
+    using Microsoft.Azure.Devices.Edge.Hub.Core;
+    using Microsoft.Azure.Devices.Edge.Hub.Core.Identity;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    public class EdgeHubTlsTransport : TlsTransport
```

Comment:
Nit: Suggest renaming this to EdgeTlsTransport to be consistent with naming elsewhere..

Reply:
Same for other EdgeHub* types

Reply:
Done. Also renamed some existing files.